{
  "id": "CVE-2020-7133",
  "assigner": "security-alert@hpe.com",
  "Published": "2020-04-24T17:15:13.41Z",
  "Modified": "2021-07-21T09:39:23.747Z",
  "last-modified": "2021-07-21T09:39:23.747Z",
  "summary": "A unauthorized remote access vulnerability was discovered in HPE IOT + GCP version(s): 1.4.0, 1.4.1, 1.4.2, 1.2.4.2.",
  "access": {
    "authentication": "NONE",
    "complexity": "LOW",
    "vector": "NETWORK"
  },
  "impact": {
    "availability": "PARTIAL",
    "confidentiality": "PARTIAL",
    "integrity": "PARTIAL"
  },
  "cvss": 7.5,
  "exploitabilityScore": 10.0,
  "impactScore": 6.4,
  "cvss-vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P",
  "references": [
    "https://support.hpe.com/hpsc/doc/public/display?docLocale=en_US&docId=emr_na-hpesbhf03947en_us"
  ],
  "vulnerable_configuration": [],
  "vulnerable_product": [],
  "vendors": [],
  "products": [],
  "vulnerable_product_stems": [],
  "vulnerable_configuration_stems": [],
  "cwe": "NVD-CWE-noinfo",
  "vulnerable_configuration_cpe_2_2": []
}